proxmox/client.go

```diff
@@ -256,6 +256,21 @@ func (c *Client) GetVmConfig(vmr *VmRef) (vmConfig map[string]interface{}, err e
 	return c.GetItemConfigMapStringInterface("/nodes/"+vmr.node+"/"+vmr.vmType+"/"+strconv.Itoa(vmr.vmId)+"/config", "vm", "CONFIG")
 }
 
+// GetStorage Get status for all datastores
+func (c *Client) GetStorage(nodeName string) (storageStatus []interface{}, err error) {
+	var data map[string]interface{}
+	url := fmt.Sprintf("/nodes/%s/storage/", nodeName)
```

Comment:
please don't use nodes and storage. 
use const of them and then put them in the line.